app/controllers/books_controller.rb

```diff
@@ -0,0 +1,21 @@
+class BooksController < ApplicationController
+  def index
+    @books = Book.all
+  end
+
+  def new
+    @book = Book.new
+  end
+
+  def create
+    @book = Book.create(book_params)
+    redirect_to books_path # persist book into the database
+    # and then redirect back to books `index`, which displays all
+    # books
```

Comment:
It looks like these comments accidentally snuck in. We try to avoid comments because they can go out of date, but code doesn't. Here's a great article Joël wrote about this topic https://robots.thoughtbot.com/letting-your-code-speak-for-itself